### DISCUSSION TITLE
print to pdf not working ?

### DISCUSSION THREAD

**Discussion by sbec101** [opengisch/QField#2794] [Q&A]
Hello everyone, I had the ' print to pdf' working earlier in the week in qfield and since the new update yesterday it's not working anymore in qfield or qfield unstable app. Is there a solution or work around. For our qfield deployment it's an important part of the qfield solution, we are migrating from esri/collector to qfield. Everyone here was super impressed with that capability.

**Comment by suricactus** [comment]
Can you share your QField and Android version, please? Tried with 2.0.15 and 2.0.16 and cannot reproduce the issue with the "Fields" layer on the Advanced bee demo project. If that demo project also works for you, can you please provide a sample project?

**Reply by sbec101** [reply]
I have an android galaxy tab active 3 , android version 11, qfield veron 2.0.15 and qfield unstable version just say master-dev (fe400e).

**Comment by nirvn** [comment]
@sbec101 , I can't replicate this issue either. Try to get us a test project from which you can't use the printing to PDF feature and attach it here. Attaching a screencast would help too.

Note that printing in 2.0 had to adapt to scoped storage so now the PDFs are written in the same folder as your project file (in the imported projects folder).

**Comment by Indiok** [comment]
I have android version 11,  installed qfield 2.5.3, and have 2 project exercise with qgis 2.28, but just one project can use menu printing to pdf that file save in qgis v 3.28
![Screenshot_20221118-215421](https://user-images.githubusercontent.com/98895632/202734033-89a632de-c43b-4940-90d6-035ed5102b5a.jpg)
![Screenshot_20221118-215055](https://user-images.githubusercontent.com/98895632/202734044-caf422be-81de-4729-9d49-204738ff6423.jpg)
Is there anything wrong with, or anything suggest for me, thanks a lot

**Reply by nirvn** [reply]
@Indiok , please attach those two sample projects, it'll facilitate debugging.

**Reply by Indiok** [reply]
thanks

**Comment by Indiok** [comment]
[GISQF.zip](https://github.com/opengisch/QField/files/10046969/GISQF.zip)

**Comment by nirvn** [comment]
@Indiok , your `Peta Banjir latihan_cloud.qgs` project has _no_ layout, so you will so _no_ layout in QField :) Proof:

![image](https://user-images.githubusercontent.com/1728657/202944762-fd13ae01-37d9-4056-b5e6-802505b751ca.png)

**Reply by Indiok** [reply]
oo... i c  Thanks a lot :D
